.papr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -xeuo pipefail
+
+export GOPATH=$HOME/gopath
+export PATH=$HOME/gopath/bin:$PATH
+export GOSRC=$HOME/gopath/src/github.com/projectatomic/libpod
```

Comment:
Hmm, is $HOME appropriate here? I dunno what "user" is running the job or where it starts from.  Maybe I'm wrong though, it's worth double checking.

Reply:
This was copied from buildah, I think in CI this will be /root